src/core/surface/completion_queue.c

```diff
@@ -45,6 +45,13 @@
 #include <grpc/support/atm.h>
 #include <grpc/support/log.h>
 
+#define MAX_PLUCKERS 4
```

Comment:
Cleaned up a little bit. I'm hesitant to add a new return code for this, so I'm abusing TIMEOUT for now.